services/install-gitlab.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+  # runs gitlab on docker
+  docker run -d --hostname 192.168.86.47 -p 443:443 -p 80:80 -p 2222:22 --name gitlab -v /srv/gitlab/config:/etc/gitlab -v /srv/gitlab/logs:/var/log/gitlab -v /srv/gitlab/data:/var/opt/gitlab --privileged ulm0/gitlab
+
+function install {
+  # create service directory
+  mkdir -p /srv/gitlab
+
+  # create yml(s)
+  {version: '3.3'
```

Comment:
I get an error when I run `services gitlab install`:
![gitlab-review-error](https://user-images.githubusercontent.com/23529006/113592521-46b0eb00-9603-11eb-8d84-73ae80222701.jpg)
Maybe related to this line not having a space (check codeclimate https://codeclimate.com/github/treehouses/cli/pull/2159).

Reply:
Fixed the Code Climate error. 😌🎉